content/pt/docs/concepts/overview/components.md

```diff
@@ -0,0 +1,118 @@
+---
+reviewers:
```

Comment:
You can probably change here to my handler :)

Reply:
Oh and add YOUR handler :) 

Like 
```
reviewers:
- pasimoes
- rikatz
``

Reply:
Not sure if those are the reviewers for the translation or the reviewers of the content.

Reply:
As far as I understand (and I did for Polish localization) you can remove all reviewers from that level of the document tree. Unless you want to assign someone special just to this part of the translated content. Just leave the team members at the top of your localization (OWNERS file).

Reply:
Remove all reviewers or ADD @rikatz?

Reply:
can remove all :) better a lot of people aware of changes in this file